src/ui/public/timepicker/kbn_global_timepicker.html

```diff
@@ -1,4 +1,4 @@
-<div ng-show="timefilter.enabled" class="kuiLocalMenu" data-test-subj="globalTimepicker">
+<div ng-show="timefilter.enabled" reporting-timefilter="{{timefilter.enabled}}" class="kuiLocalMenu" data-test-subj="globalTimepicker">
```

Comment:
We're conditionally doing some logic based on whether the timefilter is enabled or not, I could potentially reuse the `data-test-subj` attribute to find the element and see if the `ng-hide` is in the node's classList, but adding an attribute itself to tell us if it's enabled or not felt less 'brittle'.